extension-manifest-v3/README.md

```diff
@@ -8,23 +8,34 @@ This repository contains a version built with the new `chrome.declarativeNetRequ
 
 First, you need to install JavaScript dependencies and download additional resources (e.g. block lists):
 
-```sh
-$ (cd .. && npm ci)
```

Comment:
will this work correctly? specifically, will it install dependencies in `libs`?

Reply:
Yes, we have only one `node_modules` in the root path, so it goes up and install everything, which is needed for the all workspaces.

Reply:
You are right, I had to have already installed other packages. It must be done from the root.